### ISSUE TITLE
Import not supported for aws_api_gateway_vpc_link

### ISSUE THREAD

**Issue by austinbutler** [terraform-providers/terraform-provider-aws#3893]
Not sure if this is an oversight, not possible, or planned.

### Terraform Version

Terraform v0.11.5
+ provider.aws v1.11.0
+ provider.mysql v1.0.1
+ provider.template v1.0.0

### Affected Resource(s)

- aws_api_gateway_vpc_link

### Expected Behavior

API Gateway VPC link is imported.

### Actual Behavior

Terraform reports API Gateway VPC link doesn't support import.

```txt
Error: aws_api_gateway_vpc_link.staging (import id: px4ie5): import aws_api_gateway_vpc_link.staging (id: px4ie5): resource aws_api_gateway_vpc_link doesn't support import
```

### Steps to Reproduce

1. terraform import aws_api_gateway_vpc_link.staging px4ie5

**Comment by bflad** [comment]
Support for this has been merged in via #4306 and will release with v1.16.0 of the AWS provider, likely in a few days.

**Comment by bflad** [comment]
This has been released in [version 1.16.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1160-april-25-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
